bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,18 @@
+def bubbleSort(arr,lim):
+    for i in range(lim):
+        for j in range(0,lim-i-1):
+            if arr[j] > arr[j+1]:
+                arr[j],arr[j+1] = arr[j+1],arr[j]
+
+limit=int(input("Enter the limit:"))
+array=[]
+print ("Enter the numbers:")
+
+for i in range(limit):
+    temporary=int(input())
+    array.append(temporary)
+print("Unsorted array :", array)
+
+bubbleSort(array,limit)
+
+print("Sorted array :", array)    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpiizrmg4j/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpiizrmg4j/bubble_sort/Python/bubble_sort.py
@@ -15,4 +15,4 @@
 
 bubbleSort(array,limit)
 
-print("Sorted array :", array)    
+print("Sorted array :", array)
```